Dockerfile.qdrant-loader

```
@@ -0,0 +1,38 @@
+    && .venv/bin/python -m spacy download en_core_web_md
+
+
+
+FROM python:3.12-slim
+
+WORKDIR /qdrant-loader
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
+
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)